cmd/kubeadm/app/cmd/join.go

```diff
@@ -102,7 +91,21 @@ func NewCmdJoin(out io.Writer) *cobra.Command {
 
 		Often times the same token is used for both parts. In this case, the
 		--token flag can be used instead of specifying each token individually.
-		`),
+		`)
+)
+
+// NewCmdJoin returns "kubeadm join" command.
+func NewCmdJoin(out io.Writer) *cobra.Command {
+	cfg := &kubeadmapiext.NodeConfiguration{}
+	api.Scheme.Default(cfg)
+
+	var skipPreFlight bool
+	var cfgPath string
+
+	cmd := &cobra.Command{
+		Use:   "join [DiscoveryTokenAPIServers] [flags]",
```

Comment:
remove the `[DiscoveryTokenAPIServers]` here please (I know it was there before though)

Reply:
@luxas Thanks for your review, I have fix it.